From b4d0da96f9133f3f5bb4755de7015a4628a66ee5 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Tue, 10 May 2022 15:16:55 +0000
Subject: [PATCH] IWYU: add stddef.h for offsetof in sockaddr_util_posix

---
 net/base/sockaddr_util_posix.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/base/sockaddr_util_posix.cc b/net/base/sockaddr_util_posix.cc
index 80146c7..be5d303 100644
--- a/net/base/sockaddr_util_posix.cc
+++ b/net/base/sockaddr_util_posix.cc
@@ -5,6 +5,7 @@
 #include "net/base/sockaddr_util_posix.h"
 
 #include <string.h>
+#include <stddef.h>
 
 #include <sys/socket.h>
 #include <sys/un.h>
-- 
2.35.1

