From d5abab2ba80f14c11667b6fff069128d2d476b66 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Wed, 5 Jan 2022 15:23:42 +0000
Subject: [PATCH] IWYU: add atomic for std::atomic in media::MemoryDataSource

---
 media/filters/memory_data_source.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/media/filters/memory_data_source.h b/media/filters/memory_data_source.h
index 4b6b344..2a279fe 100644
--- a/media/filters/memory_data_source.h
+++ b/media/filters/memory_data_source.h
@@ -8,6 +8,7 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#include <atomic>
 #include <string>
 
 #include "base/compiler_specific.h"
-- 
2.34.1

