From e47c9e1db587de1923f7f1f946cecc397e8f2a5a Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 14 Nov 2021 10:01:21 +0000
Subject: [PATCH] IWYU: add string.h for strcmp in ui::ElementIdentifier

---
 ui/base/interaction/element_identifier.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ui/base/interaction/element_identifier.cc b/ui/base/interaction/element_identifier.cc
index 2dc0437..fe3e1f3 100644
--- a/ui/base/interaction/element_identifier.cc
+++ b/ui/base/interaction/element_identifier.cc
@@ -4,6 +4,8 @@
 
 #include "ui/base/interaction/element_identifier.h"
 
+#include <string.h>
+
 #include "base/containers/contains.h"
 #include "base/no_destructor.h"
 
-- 
2.32.0

