From 998fa82ca8e0972f9170ab1f12623b8e0b9b3911 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 16 May 2022 07:13:49 +0000
Subject: [PATCH] IWYU: add memory for std::unique_ptr in gl::GLDisplayManager

---
 ui/gl/gl_display_manager.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/gl/gl_display_manager.h b/ui/gl/gl_display_manager.h
index cbb55fc..1bdff3a 100644
--- a/ui/gl/gl_display_manager.h
+++ b/ui/gl/gl_display_manager.h
@@ -6,6 +6,7 @@
 #define UI_GL_GL_DISPLAY_MANAGER_H_
 
 #include <map>
+#include <memory>
 #include <vector>
 
 #include "base/check.h"
-- 
2.35.1

